alphagov/paas-cf
# [#145189815] Remove scripts for connecting to BOSH

## What

We've decided to move these scripts to be inline in the Team Manual, as they lack tests, may break in time and are not to be regularly used.

## How to review

* Wait until https://github.com/alphagov/paas-team-manual/pull/138 (adding the scripts to the Team Manual) is merged.
* Check that this removes all the scripts added in https://github.com/alphagov/paas-cf/pull/1023/commits/7d6ee3f65eb4fe19a9d2d4637fa0d2040b77445f, and changes nothing else.

## Who can review

Not @46bit @henrytk @timmow
